modern/src/EditItemView.js

```diff
@@ -8,18 +7,19 @@ import FormControl from '@material-ui/core/FormControl';
 
 import t from './common/localization';
 import { useEffectAsync } from './reactHelper';
+import OptionsLayout from './settings/OptionsLayout';
 
 const useStyles = makeStyles(theme => ({
   container: {
-    marginTop: theme.spacing(2),
+    marginTop: theme.spacing(2)
   },
   buttons: {
     display: 'flex',
     justifyContent: 'space-evenly',
     '& > *': {
-      flexBasis: '33%',
-    },
-  },
+      flexBasis: '33%'
```

Comment:
So far the common style was to add trailing comma.

Reply:
Actually I see you have a lot of styling changes. Can we split those into a separate PR. Otherwise it's really hard to review.

Reply:
By the way, I would prefer to keep trailing commas. It makes diffs cleaner.

Reply:
Noted